styles.css

```diff
@@ -0,0 +1,43 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  font-size: 20px;
+  background-color: rgba(26, 25, 32, 0.991);
+  color: beige;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* background-color: #333; */
+
+  /* color: white; */
+}
+
+.logo {
+  text-align: left;
+  font-size: 50px;
+  text-decoration: none;
+  color: greenyellow;
+}
+
+.bx {
+  font-size: 50px;
+  margin-right: 6vw;
+}
+
+.description {
```

Comment:
- [ ] As required for this project the section has the margins (top, bottom, left, right) defined in the design, follow the original vision, if it's pretty and works, it will be a wonderful website , kindly fix this :wink: